zzza/draw-eva.asm

```diff
@@ -18,52 +18,16 @@ draw_eva
 
 draw_sprite
 
-    ; need to draw the keyframes for moving eva along !!!
-
-    ; essentially, we re-draw hi-res buffer 4/8 times very very fast
-    ; shifting eva a tiny bit in target direction.
-    ; at end, we are done!
-
     lda     #0
     sta     FRAMES_SINCE_MOVE
 
     ; calculate direction for animation
     jsr     update_sprite_direction
 
-; quick animation to draw EVA moving without having to worry about actually syncing it with the level
-draw_quick_loop
-
-    jsr     reset_high_res              ; clear high res graphics
-    
-    ; once the direction is set...
-    ; draw EVA at the appropriate position in buffer
-    jsr     draw_shift_horizontal       ; draw the appropriate shift left/right
-    jsr     draw_shift_vertical         ; draw the appropriate shift up/down
-
-    jsr     mask_level_onto_hi_res      ; once EVA is in correct position, fill in the level from adjacent level data 
-    
-    jsr     draw_high_res               ; draw high-res buffer to EVA's position on the screen
```

Comment:
I think what was happening here is that we were doing all the hi-res operations twice. Once here, and once in `draw_master_hi_res`.